bin/cliain/Cargo.toml

```diff
@@ -6,21 +6,34 @@ edition = "2021"
 [dependencies]
```

Comment:
uhm, I do not know whether we should bump the version or not...

Reply:
bumped just in case